app/src/main/java/com/example/footprnt/Map/Util/MapUtil.java

```diff
@@ -179,4 +197,149 @@ public static JSONObject getContinents(Activity activity){
         return mContinents;
     }
 
+    /**
+     * On search result, locates place on map
+     *
+     * @param searchText
+     * @param map
+     * @param context
+     */
+    public static void geoLocate(EditText searchText, GoogleMap map, Context context){
+        String searchString = searchText.getText().toString();
+
+        Geocoder geocoder = new Geocoder(context);
+        List<Address> list = new ArrayList<>();
+        try{
+            list = geocoder.getFromLocationName(searchString, 1);
+        } catch (IOException e){
+
+        }
+
+        if (list.size() > 0){
+            Address address = list.get(0);
+            Location l = new Location(LocationManager.GPS_PROVIDER);
+            l.setLatitude(address.getLatitude());
+            l.setLongitude(address.getLongitude());
+            centreMapOnLocation(map, l);
+        }
+    }
+
+    /**
+     * Centres camera on given location
+     *
+     * @param map
+     * @param location
+     */
+    public static void centreMapOnLocation(GoogleMap map, Location location) {
+        LatLng userLocation = new LatLng(location.getLatitude(), location.getLongitude());
+        map.animateCamera(CameraUpdateFactory.newLatLngZoom(userLocation, 12));
+    }
+
+    /**
+     * Creates post and sends to Parse server
+     *
+     * @param activity
+     * @param context
+     * @param description
+     * @param title
+     * @param imageFile
+     * @param user
+     * @param point
+     * @param tags
+     * @return
+     */
+    public static Post createPost(Activity activity, final Context context, String description, String title, ParseFile imageFile, ParseUser user, LatLng point, ArrayList<String> tags) {
+        final Post newPost = new Post();
+        newPost.setDescription(description);
+        if (imageFile == null) {
+            newPost.remove(com.example.footprnt.Util.Constants.image);
```

Comment:
Can we import `com.example.footprnt.Util.Constants` and just `Constants.image` here? Also static values like this should be named `IMAGE` not `image`